requirements/requirements-optionals.txt

```diff
@@ -2,3 +2,4 @@
 markdown==2.5.2
 django-guardian==1.3.2
 django-filter==0.10.0
```

Comment:
Ah, yes. The output is quite... different, if pytz is not installed. The test results end up looking like:

    AssertionError: '"<django.utils.timezone.LocalTimezone object at 0x105a5af90>"' != '"America/Los_Angeles"'